docs/scos/user/back-office-user-guides/202200.0/merchandising/discount/creating-vouchers.md

```diff
@@ -303,44 +306,44 @@ This section contains a set of tables that describe fields, value types, and ope
 | VALUE | DESCRIPTION |
 | --- | --- |
 | Fields | The available fields may include SKU, item-price, item-quantity, or a variety of attributes (for example, **currency** on the image above). |
-| Operator | Operator compares the value of a field on the left with the value(s) on the right (for example, equals (=), greater than (>)). If the expression evaluates to true, the discount can be applied. (operator is **equal** on the image above) |
-| Value | Value types must match the selected field. The asterisk (*) matches all possible values. (on the image above, the value is **Swiss Franc**)|
-| Combine Conditions | ‘AND’ and ‘OR’ operators are used to combine conditions. (**AND** on the image above) |
-|Grouping|When building more complex queries, conditions may be grouped inside parentheses ‘( )’.|
+| Operator | Operator compares the value of a field on the left with the value(s) on the right (for example, equals (=), greater than (>)). If the expression evaluates to true, the discount can be applied (operator is **equal** on the preceding image). |
+| Value | Value types must match the selected field. The asterisk (*) matches all possible values (on the preceding image, the value is **Swiss Franc**).|
+| Combine Conditions | 'AND' and 'OR' operators are used to combine conditions (**AND** on the preceding image). |
+|Grouping | When building more complex queries, conditions may be grouped inside parentheses '( )'.|
```

Comment:
```suggestion
|Grouping | When building more complex queries, conditions may be grouped inside parentheses '( )'. Because discount calculations and conditions are applied per item, it is not possible to use groups with AND that each contain SKU parameters. |
```